src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -510,11 +510,14 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\UnmanagedType.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\VarEnum.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector64.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector64DebugView.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector64`1.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector64DebugView`1.cs" />
```

Comment:
Is it necessary to rename these with back-ticks? The `Vector<T>` equivalent are called Vector.cs and Vector_Operations.cs.

Reply:
It's not necessary, but we don't seem to have much consistency here. Some are name `_1`, Vector uses `_Operations`, etc. I went with backtick since it matches the actual name of the type.

Reply:
Well, FWIW I'm not a fan of the back-tick.

Reply:
We got rid of backticks in filenames because they break stuff: https://github.com/dotnet/corefx/issues/8608. We should not be using them.

Reply:
Fixed by replacing it with `_`, as that seems to be the convention used to fix the backtick case more generally.